drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/editor/ScenarioSimulationEditorPresenter.java

```diff
@@ -392,6 +409,34 @@ protected void open(final String downloadURL) {
         DomGlobal.window.open(downloadURL);
     }
 
+    protected void showImportDialog() {
+        eventBus.fireEvent(new ImportEvent());
+    }
+
+    protected void onExportToCsv() {
+        importExportService.call(getExportCallBack(),
+                                 new DefaultErrorCallback())
+                .exportSimulation(CSV, context.getStatus().getSimulation());
+    }
+
+    protected RemoteCallback<Object> getExportCallBack() {
+        return rawResult -> {
+            final AnchorElement exportAnchorElement = view.getExportAnchorElement((String) rawResult, path.getFileName() + CSV.getExtension());
+            final NativeEvent CLICK_EVENT = Document.get().createClickEvent(0, 0, 0, 0, 0, false, false, false, false);
```

Comment:
Just wondering. What is this used for?

Reply:
No more used :D
Btw I used this _trick_ to force client side file creation and download. Now I found a better way used by Stunner too and I updated the code

Reply:
Ok. Thought this was something like that.